src/preferences.jl

```diff
@@ -0,0 +1,17 @@
+import Dagger
+import Preferences
+
+function set_distributed_package!(package_name::String)
+    if package_name ∉ ("Distributed", "DistributedNext")
+        throw(ArgumentError("Invalid distributed package: $(package_name)"))
+    end
+    Preferences.set_preferences!("Dagger", "distributed-package" => package_name;
+                                 force = true)
+    Preferences.set_preferences!("MemPool", "distributed-package" => package_name;
+                                 force = true)
+    Preferences.set_preferences!("TimespanLogging", "distributed-package" => package_name;
+                                 force = true)
```

Comment:
```suggestion
    Dagger.set_distributed_package!(package_name)
```

This can be simplified once TimespanLogging is bumped JuliaParallel/Dagger.jl#597